### DISCUSSION TITLE
Add new software/url to list

### DISCUSSION THREAD

**Discussion by viulian77** [microsoft/winget-cli#1394] [General] [answered]
Hi,

  Is it posible to configure winget to add extra software/url (not listed in winget show) to the list of programs that can be upgraded/installed ? How ?

 Thank you

**Comment by ImJoakim** [comment, marked as answer]
You can submit a package to the [Windows Package Manager Community Repository](https://github.com/microsoft/winget-pkgs), which will then show up in `winget show` and you will be able to install the software. You can read more about the steps to submit a package here: [Submit a Package](https://github.com/microsoft/winget-pkgs#submitting-a-package).

Also, if you want someone else to create the manifest and add the software for you, please fill out [Package Request/Submission](https://github.com/microsoft/winget-pkgs/issues/new?assignees=&labels=Help-Wanted%2C+Package-Request&template=Manifest_Request.md&title=) with as much information as possible.

Hope this answered your questions 😃

**Reply by ImJoakim** [reply]
> I was thinking more in a particular way, by adding a "private Repo", availeble only for that user/company.

Ah I see, my apologies. I believe the following issue/discussion may be related to your question:

* https://github.com/microsoft/winget-cli/issues/226

**Comment by viulian77** [comment]
Yes. Thanks.
